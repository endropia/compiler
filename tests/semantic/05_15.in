var
	a: boolean;
	b: string;
begin
	writeln(a = b);
end.