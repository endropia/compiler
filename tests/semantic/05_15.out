(5, 12) EQUAL operation is not overloaded for boolean and string